### DISCUSSION TITLE
Having no last name is creating an issue

### DISCUSSION THREAD

**Discussion by jgyasu** [community/community#121658] [GitHub Education]
Hello, I recently applied for the GitHub education pack and I don't have a last name, it's completely legal in my country. 

While entering my information on the billing page, they mandatorily asked for a last name so I put something random as my last name but my official identities and college admission details don't have my last name and as a result the verification failed.

What should I do? Please help

**Comment by samus-aran** [comment]
Hi @jgyasu,

If you have a family name use that as your last name. 
If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

**Reply by jgyasu** [reply]
I only have a single legal name and that is what is mentioned in my college documents. If I fill it in both the first and last name, it will be different from my documents, won't it cause any issues?
